source/core/Variants.cpp

```diff
@@ -67,6 +68,17 @@ TypeRef CopyToVariant(TypeRef sourceType)
     return variant;
 }
 
+bool IsTypeCompatibleWithInput(TypeRef inputType, TypeRef destType, void* destData)
+{
+    // If type terminal is unwired, or it is the same as the input, return true.
+    return inputType && (!destData || inputType->IsA(destType, true));
+}
+
+bool IsTerminalUnwiredOrVariant(TypeRef destType, void* destData)
```

Comment:
It is better to name functions and explain comments in VIA parlance first whenever we can (and use G terminology in addition to that in comments or instead of when necessary). In stead of "terminal unwired", it might be better to say "if optional argument is null or not specified or something to that effect"

Reply:
Yeah, I was conflicted on my use of "terminal". I will change this.